src/utils/results-reporter.ts

```diff
@@ -85,33 +87,57 @@ export class ResultsReporter {
     const screensWithDifferences = screenshotDiffResults.filter(
       (result) => result.outcome === "diff"
     ).length;
-    const totalScreens = screenshotDiffResults.length;
+    const totalScreensCompared = screenshotDiffResults.length;
+    const totalScreenshotsTaken = testCaseResults.reduce(
+      (total, testCase) => total + testCase.totalNumberOfScreenshots,
+      0
+    );
 
     if (screensWithDifferences === 0) {
       if (!(testRun.project as EnrichedProject).isGitHubIntegrationActive) {
         await this.setCommitStatus({
-          description: `Zero differences across ${totalScreens} screens tested`,
+          description: `Zero differences across ${totalScreensCompared} screens tested`,
           state: "success",
           targetUrl: testRun.url,
         });
       }
-      if (totalScreens > 0) {
+      if (totalScreensCompared > 0) {
         await this.setStatusComment({
           createIfDoesNotExist: true,
-          body: `✅ Meticulous spotted zero visual differences across ${totalScreens} screens tested: [view results](${testRun.url}).`,
+          body: `✅ Meticulous spotted zero visual differences across ${totalScreensCompared} screens tested: [view results](${testRun.url}).`,
         });
+      } else {
+        if (totalScreenshotsTaken === 0) {
+          if (testCaseResults)
```

Comment:
[nit] I think this is the first time we've used a bracket-less if statement. Thinking maybe we avoid for readability?

Reply:
Also maybe `if (testCaseResults.length > 0)` would be more explicit? (presume it can't be null otherwise https://github.com/alwaysmeticulous/report-diffs-action/pull/220/files#diff-8ab64f1b06c30c9fe0c1d52618066179d30779973cd505455ed019e68add5bdaR91 would crash?)

Reply:
oops that was unintentional https://github.com/alwaysmeticulous/report-diffs-action/pull/220/commits/ef770fd2b44cdc58023a2b7bcdc5c567917ee32b drops it. I think It being 0 would also result a crash